beacon-chain/core/validators/validator.go

```diff
@@ -179,7 +180,14 @@ func SlashValidator(
 	); err != nil {
 		return nil, err
 	}
-	if err := helpers.DecreaseBalance(s, slashedIdx, validator.EffectiveBalance/penaltyQuotient); err != nil {
+
+	slashingQuotient, proposerRewardQuotient, whistleblowerRewardQuotient, err := SlashingParamsPerVersion(s.Version())
+	if err != nil {
+		return nil, errors.Wrap(err, "could not get slashing parameters per version")
+	}
+
+	slashingPenalty := validator.EffectiveBalance / slashingQuotient
```

Comment:
safe math here? just to be careful

Reply:
Done in 64dcaa8f4a52 for all divisions in this method